spec/views/defence_requests/new.html.erb_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
```

Comment:
I think it's a good idea to use a view spec to test this. We don't have any view specs though, so I think it should be agreed by the wider team.